openssl/src/x509/mod.rs

```diff
@@ -842,6 +842,10 @@ impl X509Extension {
     /// mini-language that can read arbitrary files.
     ///
     /// See the extension module for builder types which will construct certain common extensions.
+    ///
+    /// This function is deprecated, `X509Extension::new_from_der` or the
+    /// types in `x509::extension` should be used in its place.
+    #[deprecated]
```

Comment:
We should add `note` and `since` sections to these: https://github.com/sfackler/rust-openssl/blob/47abce45f95dbca9c644674abe40d60ede01dc64/openssl/src/pkcs12.rs#L111